### PR TITLE
Updated packages and fixed associated run-time errors

### PR DESCRIPTION
**Pull request by tom-schier**

### CODE REVIEW

File: LICENSE [hunk 1]
```diff
@@ -1,21 +0,0 @@
-MIT License
```
Comment (perlboy) [inline LICENSE:1]: @tom-schier Deleting the license?

File: package.json [hunk 2]
```diff
@@ -20,8 +20,8 @@
     "flat": "^5.0.0",
     "html-entities": "^1.2.1",
     "http-proxy": "^1.18.0",
-    "jose": "^1.10.1",
-    "lodash": "^4.17.20",
```
Comment (perlboy) [inline package.json:24]: Depandabot build for `lodash` update previously failed?
 https://github.com/Regional-Australia-Bank/ADR-Gateway/pull/45

File: package.json [hunk 3]
```diff
@@ -84,12 +84,12 @@
     "cpx": "^1.5.0",
     "cross-env": "^7.0.2",
     "get-port": "^5.1.1",
-    "handlebars": "^4.7.6",
```
Comment (perlboy) [inline package.json:87]: Closes #41

File: package.json [hunk 4]
```diff
@@ -20,8 +20,8 @@
     "flat": "^5.0.0",
     "html-entities": "^1.2.1",
     "http-proxy": "^1.18.0",
-    "jose": "^1.10.1",
```
Comment (perlboy) [inline package.json:23]: Closes #37

File: package.json [hunk 5]
```diff
@@ -33,7 +33,7 @@
     "typeorm": "^0.2.19",
     "typescript": "3.7.x",
     "url-join": "^4.0.1",
-    "url-parse": "^1.4.7",
```
Comment (perlboy) [inline package.json:36]: Build in #42 for 1.5.0 of this change previously failed?

File: src/Common/Connectivity/Dependencies.generator.js [hunk 6]
```diff
@@ -6,7 +6,7 @@ const _ = require("lodash")
 const dependenciesTemplate = Handlebars.compile(fs.readFileSync("src/Common/Connectivity/DependencyGraph.template.hbs","utf8"))
 const connectorTemplate = Handlebars.compile(fs.readFileSync("src/Common/Connectivity/Connector.template.hbs","utf8"))
 
-let dependencies = yaml.safeLoad(fs.readFileSync("src/Common/Connectivity/Dependencies.yml","utf8"));
+let dependencies = yaml.load(fs.readFileSync("src/Common/Connectivity/Dependencies.yml","utf8"));
```
Comment (perlboy) [inline src/Common/Connectivity/Dependencies.generator.js:9]: Appears to rely on a recent `js-yaml` but the version is not explicitly stated within package.json?

File: src/MockServices/DhServer/Server/Handlers/GrantConsent.ts [hunk 7]
```diff
@@ -2,7 +2,7 @@ import { Dictionary } from "../../../../Common/Server/Types";
 import _ from "lodash";
 import express from "express";
 import { NextFunction } from "connect";
-import { getType } from "mime";
+
```
Comment (perlboy) [inline src/MockServices/DhServer/Server/Handlers/GrantConsent.ts:5]: Might as well delete the line...

File: src/MockServices/DhServer/Server/Handlers/UpdateClientRegistration.ts [hunk 8]
```diff
@@ -1,7 +1,7 @@
 import _ from "lodash";
 import express from "express";
 import { NextFunction } from "connect";
-import { getType } from "mime";
+
```
Comment (perlboy) [inline src/MockServices/DhServer/Server/Handlers/UpdateClientRegistration.ts:4]: Might as well delete the line...

**Review by perlboy — CHANGES_REQUESTED** [review]

File: src/Common/Connectivity/Dependencies.generator.js (hunk 6 above)
Comment (pcurtisrab) [reply src/Common/Connectivity/Dependencies.generator.js:9]: Still not sure about this one. I'll be pushing a new commit to take care of some of the feedback from this PR in a moment.

What is the problem with safeLoad? Noting that the functionality is the same in js-yaml@4, but js-yaml@3 appears to be installed. Due to #40, right now I'm thinking to leave this like it is.

File: package.json (hunk 4 above)
Comment (pcurtisrab) [reply package.json:23]: The pm2 deployment breaks with the new major version of convict. Might need to leave this for a future PR if there is some functionality in v6 that is desired.

**Comment by pcurtisrab** [comment]
Thanks for the PR, @tom-schier , and your input @perlboy.

I've had to pick and choose a bit from this PR, so what I have taken from it is in https://github.com/Regional-Australia-Bank/ADR-Gateway/commit/c635bfea64aeb8644d0ab86c35ea79858595afcd.

What I haven't taken, I have commented on above. Please raise a new PR for related changes.

**Pull request closed without merging**